applications/crossbar/src/modules/cb_webhooks.erl

```diff
@@ -356,6 +394,51 @@ summary_available(Context) ->
               ],
     crossbar_view:load(cb_context:set_account_db(C1, MasterAccountDb), ?AVAILABLE_HOOKS, Options).
 
+%%------------------------------------------------------------------------------
+%% @doc
+%% @end
+%%------------------------------------------------------------------------------
+-spec fetch_webhook_samples(cb_context:context(), kz_term:ne_binary()) -> cb_context:context().
+fetch_webhook_samples(Context, WebhookName) ->
+    case fetch_webhook_samples(WebhookName) of
+        {'ok', JObjs} ->
+            crossbar_doc:handle_json_success(JObjs, Context);
+        {'error', 'not_found'} ->
+            crossbar_util:response_bad_identifier(WebhookName, Context);
+        {'error', _} ->
+            cb_context:add_system_error('datastore_fault', Context)
+    end.
+
+-spec fetch_webhook_samples(kz_term:ne_binary() | kz_term:ne_binaries()) ->
+                                   {'ok', kz_json:objects()} |
+                                   {'error', any()}.
+fetch_webhook_samples(<<"webhooks_all">>) ->
```

Comment:
I think having to fetch `/v2/.../webhooks/samples/all` is a bit strange vs listing available samples via `/v2/.../webhooks/samples/` then querying the specific sample you want to see via `/webhooks/samples/{SAMPLE_ID}`.